bin/run.sh

```diff
@@ -28,12 +28,7 @@ if [ -z "$slug" ] || [ -z "$solution_path" ]; then
 fi
 
 cd "$solution_path"
-if [ -e Cargo.lock ]; then
-    if [ "$(grep -c '\[\[package\]\]' Cargo.lock)" -gt 1 ]; then
-        echo "{\"status\":\"error\",\"message\":\"building $slug: external crates not supported\",\"tests\":[]}" > "$output_path"/results.json
-        exit
-    fi
-else
+if [ ! -e Cargo.lock ]; then
```

Comment:
It would be nice if at some later point there would be an error messageif the student uses a non-supported crate.

Reply:
Yes, I was thinking about this. I'll think of a way to support this.